modules/yardstick/src/main/java/org/apache/ignite/yardstick/cache/load/IgniteCacheRandomOperationBenchmark.java

```diff
@@ -117,18 +139,36 @@
     }
 
     /** {@inheritDoc} */
+    @Override public void onException(Throwable e) {
+        BenchmarkUtils.errorHelp(cfg, "The benchmark of random operation failed.");
+        super.onException(e);
+    }
+
+    /** {@inheritDoc} */
     @Override public boolean test(Map<Object, Object> map) throws Exception {
         if (nextBoolean()) {
-            executeInTransaction();
+            executeInTransaction(map);
 
-            executeOutOfTx(true);
+            executeOutOfTx(map, true);
         }
         else
-            executeOutOfTx(false);
+            executeOutOfTx(map, false);
 
         return true;
     }
 
+    /** {@inheritDoc} */
+    @Override public void tearDown() throws Exception {
+        BenchmarkUtils.println("Benchmark statistics");
+        for (String cacheName : ignite().cacheNames()) {
+            BenchmarkUtils.println(String.format("Operations over cache '%s'", cacheName));
+            for (Operation op : Operation.values())
```

Comment:
Don't stick every line of the code to each other. You have to put a carriage return (new line) in between these two lines. Please take a look one more time on our coding guidelines and see how the guidelines are applied across the sources. For simplicity you can take a look at how the coding guidelines are followed by other benchmarks.